kt/godot-runtime/src/main/kotlin/godot/core/Functions.kt

```diff
@@ -58,7 +58,7 @@ abstract class KtFunction<T : KtObject, R : Any?>(
         val functionInfo: KtFunctionInfo,
         val parameterCount: Int,
         val variantType: VariantType,
-        private vararg val parameterTypes: VariantType
+        private vararg val parameterTypes: Pair<VariantType, Boolean>
```

Comment:
I would rather use it only as a constructor argument to initialize two array of primitives. To avoid an array of complex object in buffer context.